tests/integration/components/main-menu-test.js

```diff
@@ -0,0 +1,28 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | main-menu', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders title and all links', async function (assert) {
```

Comment:
when the test has more than one assertion in Selleo we add `assert.expect(numberOfAssertion)`